src/osd/PG.cc

```diff
@@ -1336,8 +1336,11 @@ bool PG::sched_scrub()
 	  << (is_active() ? ") <active>" : ") <not-active>")
 	  << (is_clean() ? " <clean>" : " <not-clean>") << dendl;
   ceph_assert(ceph_mutex_is_locked(_lock));
-  ceph_assert(!is_scrubbing());
```

Comment:
@ronen-fr are you able to reproduce this assert? Is the fuller fix in Q included in one of your existing PRs, in other words, can we reference a commit in this PR's description?

Reply:
@neha-ojha:
> are you able to reproduce this assert? 
I have never encountered it during standalone testing.

> Is the fuller fix in Q included in one of your existing PRs, in other words, can we reference a commit in this PR's description?

(to be completed)

Reply:
@ronen-fr The reason I asked was because ideally we want to get the fix merged in master first and then backport to pacific, to avoid any regressions. In this case, since we have a minimal fix going into pacific, can we reference the master commit, for future reference?

Reply:
Ah. Got it.
I think the best way is therefore to create the same fix for master.
Doing this now.